src/main.rs

```diff
@@ -21,14 +29,19 @@ fn main() -> anyhow::Result<()> {
                     continue;
                 }
 
-                items.push(format!(
-                    "{} = {}",
-                    entry.path().to_string_lossy().to_string(),
-                    repo.head()?.name().unwrap_or("None")
-                ));
+                let head = repo.head()?;
+                if let Some(head) = head.name() {
+                    repos.push(PersistableRepo {
+                        // Ideally we wanna do this, but it moves `dir`.
+                        // path: entry.path().to_string_lossy().strip_prefix(dir).unwrap().to_string(),
+                        path: entry.path().to_string_lossy().to_string(),
+                        remote_url: repo.remotes()?.get(0).unwrap_or("None").to_string(),
```

Comment:
Is this the optimal remote to use?

Reply:
Nope, we ideally want to use the upstream remote for the currently checked out branch. I haven't figured out how to do that yet.